momepy/functional/_diversity.py

```diff
@@ -176,6 +188,322 @@ def describe(
     return _compute_stats(grouper, q, include_mode)
 
 
+def values_range(
+    data: DataFrame | Series, graph: Graph, rng: tuple | list = (0, 100), **kwargs
```

Comment:
```suggestion
    y: DataFrame | Series | NDArray[np.float_], graph: Graph, q: tuple | list = (0, 100), **kwargs
```
rng is now used by numpy for random-number-generator. Let's switch as we did in `describe`. And let's call the input array `y` as we do in desribe.